src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests.cs

```diff
@@ -2711,6 +2711,72 @@ void M(object o)
                 expectedIndentation: 16);
         }
 
+        [WpfFact]
+        [WorkItem(33253, "https://github.com/dotnet/roslyn/issues/33253")]
+        [Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        public void EnterAfterFluentSequences_1()
+        {
+            var code = @"public class Test
+{
+    public void Test()
+    {
+        new List<DateTime>()
+            .Where(d => d.Kind == DateTimeKind.Local ||
```

Comment:
❓  Can you add one more test (keeping all the current ones) which is like this one, but has the `.Where` indented twice instead of once? In this case, the expected indentation would end up as 16.

Reply:
Yeah, sure, added. Any particular reason for this specific test case?

Reply:
All the other test cases were indented one level relative to the parent statement. The new test ensures that we are matching the indentation of the previous line for this case.